lisp/ein-utils.el

```diff
@@ -658,18 +659,26 @@ Use `ein:log' for debugging and logging."
       (ein:display-warning message level)
       (puthash key t ein:display-warning-once--db))))
 
-(defun ein:get-docstring (function)
+(defun ein:get-docstring (func)
   "Return docstring of FUNCTION."
   ;; Borrowed from `ac-symbol-documentation'.
-  (with-temp-buffer
-    ;; import help-xref-following
-    (require 'help-mode)
-    (erase-buffer)
-    (let ((standard-output (current-buffer))
-          (help-xref-following t)
-          (major-mode 'help-mode)) ; avoid error in Emacs 24
-      (describe-function-1 function))
-    (buffer-string)))
+  (let ((f func))
+    (deferred:$
```

Comment:
`ein:get-docstring` will now return whatever `deferred:$` returns, which is not the docstring (it won't have it right away).  The upshot is mouseovers of menu items no longer show the docstring, which I personally don't mind as I never use menus, but probably not what you intended.

Reply:
Ugh, I hate deferred. Thanks - I can't really prove if this is helping performance or not anyway, so I'll just shelve it for the time being.